src/pages/home.tsx

```diff
@@ -0,0 +1,46 @@
+import { type FC, type PropsWithChildren, useState } from "react";
+import NavBar from "~/components/ui/NavBar";
+
+const INITIAL_PAGE = 1;
+
+type PageContentProps = {
+  page: number;
+  index: number;
+};
+
+const PageContent: FC<PropsWithChildren<PageContentProps>> = ({
+  children,
+  page,
+  index,
+}) => {
+  return (
+    <main className={`${page === index ? "" : "hidden"}`}>{children}</main>
+  );
```

Comment:
just don't render the element instead of using CSS `display: none`

https://stackoverflow.com/questions/28376180/is-it-better-to-not-render-html-at-all-or-add-displaynone

```suggestion
}) => {
  if (page !== index) {
      return <></>;
  }

  return (
    <main>{children}</main>
  );
```